cg/services/quality_controller/quality_checks/checks.py

```diff
@@ -0,0 +1,62 @@
+from typing import Callable
+from enum import Enum
+
+from cg.constants import Workflow
+from cg.store.models import Case
+from cg.services.quality_controller.quality_checks.utils import (
+    case_pass_sequencing_qc,
+    sample_pass_sequencing_qc,
+    any_sample_in_case_has_reads,
+)
+
+
+class QualityChecks(Enum):
+    """
+    Parent class for the quality checks.
+    """
+
+
+class SequencingQC(QualityChecks):
```

Comment:
I'm not sure if the parent class is necessary

```suggestion
class SequencingQCChecks(QualityChecks):
```